app/models/comment.rb

```diff
@@ -1,4 +1,14 @@
 class Comment < ApplicationRecord
   belongs_to :user
```

Comment:
- [OPTIONAL ] I think it would be great if you add the class name beside user `, class_name: 'User'` to have the association well so in the future not face problems with it ✅🔥